teams/v1.12/MicrosoftTeams.schema.json

```diff
@@ -0,0 +1,1073 @@
+                "maximum": 50
+              }
+            },
+            "required": [ "id", "name", "file", "preview", "maxAudience", "seatsReservedForOrganizersOrPresenters" ]
+          },
+          "maxItems": 5,
+          "type": "array",
+          "uniqueItems": true
+        },
+        "filters": {
+          "description": "Meeting supported A/V filters.",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "id": {
+                "$ref": "#/definitions/guid",
+                "description": "A unique identifier for this A/V filter. This id must be a GUID."
+              },
+              "name": {
+                "type": "string",
+                "description": "A/V filter's name.",
+                "maxLength": 128
+              },
+              "thumbnail": {
+                "$ref": "#/definitions/relativePath",
+                "description": "A relative file path to a A/V filter's thumbnail."
+              }
+            },
+            "required": [ "id", "name", "thumbnail" ]
+          },
+          "maxItems": 100,
+          "type": "array",
+          "uniqueItems": true
+        },
+        "videoAppContentUrl": {
+          "type": "string",
+          "description": "The url which indicates A/V filters' video app content.",
+          "maxLength": 2048
+        }
+      },
+      "description": "Specify meeting extension definition.",
+      "additionalProperties": false
+    },
+    "authorization": {
+      "type": "object",
+      "description": "Specify and consolidates authorization related information for the App.",
+      "additionalProperties": false,
+      "properties": {
+        "permissions": {
+          "type": "object",
+          "description": "Specify Permissions which App lists for accessing Resource.",
+          "additionalProperties": false,
+          "properties": {
+            "orgWide": {
+              "description": "App's organization wide classic permissions provided by user.",
+              "maxItems": 16,
+              "type": "array",
+              "uniqueItems": true,
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "name": {
+                    "type": "string",
+                    "description": "The Permission name of app's permission node.",
+                    "maxLength": 128
+                  },
+                  "type": {
+                    "type": "string",
+                    "enum": [
+                      "Application",
+                      "Delegated"
+                    ],
+                    "description": "The Permission type of app's permission node."
+                  }
+                },
+                "required": [
+                  "name",
+                  "type"
```

Comment:
We are not ready to support org-wide permissions yet. These should not be part of v1.12

Reply:
Removed